src/Compute/Compute/Generated/DedicatedHostGroup/DedicatedHostGroupGetMethod.cs

```diff
@@ -121,6 +132,12 @@ public override void ExecuteCmdlet()
         [ResourceNameCompleter("Microsoft.Compute/hostGroups", "ResourceGroupName")]
         public string Name { get; set; }
 
+        [Parameter(
+            ParameterSetName = "DefaultParameter",
```

Comment:
It would be better if gives some help messages

Reply:
The common practice is to write parameter description (aka help message) in the `[Parameter()]` attribute. When [generating help](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md) it will be written to the markdown files.

However it's also OK (though not recomended) to manually update the help file.

Reply:
Understood. Thank you.

Reply:
I will make some updates and submit a PR fixing that